### PR TITLE
feat: [#1067] url box redirect to KH website

### PR DESCRIPTION
**Pull request by Anok666**

### CODE REVIEW

**Comment by aleksandrahojszyk** [comment]
approve

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html [hunk 1]
```diff
@@ -89,6 +89,13 @@ <h5>No results</h5>
         "
       ></ess-sort-by-functionality>
     </div>
+   <ng-container  *ngIf="this.type === 'training'">
+    <a href="https://knowledge-hub.eosc-portal.eu">
```
Comment (maria-j-k) [inline ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html:93]: Shouldn't we take the url from the config?

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html [hunk 2]
```diff
@@ -89,6 +89,13 @@ <h5>No results</h5>
         "
       ></ess-sort-by-functionality>
     </div>
+   <ng-container  *ngIf="this.type === 'training'">
```
Comment (maria-j-k) [inline ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html:92]: Why this condition is based on the type of the first result? 
If all_collection would have a training item as the first result the banner should appear?
We can get the collection from the `CustomRoute`

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html (hunk 1 above)
Comment (Anok666) [reply ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html]: true, done.

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html (hunk 2 above)
Comment (Anok666) [reply ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html]: this box with url is only for the training collection. However, I based it on CustomRoute and collections after your comment

**Review by Marcelinna — APPROVED** [review]

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts [hunk 3]
```diff
@@ -42,6 +43,8 @@ export class ResultsWithPaginationComponent implements OnInit {
       this.isError = false;
     }
 
+    this.type = response.results[0].type.value;
```
Comment (maria-j-k) [inline ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts:46]: is this still needed?

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts [hunk 4]
```diff
@@ -30,6 +30,7 @@ export class ResultsWithPaginationComponent implements OnInit {
   } = {};
   isError = false;
   marketplaceUrl = '';
+  type = '';
```
Comment (maria-j-k) [inline ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts:33]: is this still needed?

**Review by maria-j-k — APPROVED** [review]

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts (hunk 4 above)
Comment (Anok666) [reply ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts]: done

File: ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts (hunk 3 above)
Comment (Anok666) [reply ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts]: done

**Pull request merged**